Resources/Prototypes/_White/Recipes/hidden_crafts.yml

```
@@ -0,0 +1,11 @@
+- type: constructionGraph
+  id: EnergyDoubleSwordGraph
+  start: esword
+  graph:
+  - node: esword
+    edges:
+      - to: desword
+        steps:
+          - tag: EnergySword
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)